### ISSUE TITLE
Line breaks immediately followed by ANSI escapes are swallowed

### ISSUE THREAD

**Issue by bruderstein** [wallabyjs/public#262]
Using [unexpected](https://github.com/unexpectedjs/unexpected), and the new ANSI escape sequence parsing in wallaby (issue #237), wallaby displays the full colour error message.

However, when a newline is immediately followed by an escape code, the newline is swallowed.

Compare:
![uenxpected_break](https://cloud.githubusercontent.com/assets/91716/9837328/5e1538e6-5a3b-11e5-9cf1-b91664095ef6.png)

with the same test run on the console (unexpected is actually intellegent about the line breaks, so this is on a narrower console, if the console is wider, there's just a space, not sure if that has something to do with it, but the mock tests in the demo below would suggest not)

![console_break](https://cloud.githubusercontent.com/assets/91716/9837325/551a0230-5a3b-11e5-9ccf-fa8afc51ec82.png)

I've mocked out the `UnexpectedError` object to try and work out exactly when they're swallowed, and it appears to be when an escape code immediately follows the new line. 

A demo is at [bruderstein/wallaby-ansi-parser](https://github.com/bruderstein/wallaby-ansi-parser)

One "real" test is included, where the difference between running `npm test` in the console and running it in wallaby can be seen.

3 small "mock" tests that only really work in wallaby also show the difference a single space makes following the line break.

```js
   it('shows line breaks when lines begin with letter', function () {
       throw new UnexpectedError('\x1b[32mLine one\nLine two\x1b[39m');
   });

    it('swallows line breaks when lines begin with ansi code', function () {
        throw new UnexpectedError('\x1b[32mLine one\nLine two\x1b[39m\n\x1b[31mLine 3\x1b[39m');
    });

    it('a simple space is enough to keep the line break', function () {
        throw new UnexpectedError('\x1b[32mLine one\nLine two\x1b[39m\n \x1b[31mLine 3\x1b[39m');
    });
```
![mocktests](https://cloud.githubusercontent.com/assets/91716/9837345/cbdb4384-5a3b-11e5-893b-18e46484b6ad.png)

Wallaby IntelliJ plugin 1.0.28
Wallaby core 1.0.104
WebStorm 11 (142.4148), Linux

Thank you :)

**Comment by ArtemGovorov** [comment]
Thanks a lot for the demo repo and the analysis on when exactly the issue is reproduced!
It's now fixed in Wallaby IntelliJ plugin v1.0.29.

**Comment by bruderstein** [comment]
Wow, thanks!
